n4js-libs/packages/n4js-cli/package.json

```diff
@@ -7,7 +7,7 @@
   "internalVersion": "2019-03-15T08:46:12Z",
   "description": "N4JS compiler and command line tools.",
   "scripts": {
-    "postinstall": "./npm-postinstall.js",
+    "postinstall": "node ./npm-postinstall.js",
     "build": "./npm-build.sh",
```

Comment:
And what about this shell script `npm-build.sh`? Does it work on Windows?
Also `n4jsc.js` a few lines further down: doesn't that one need to be adjusted, as well?

Reply:
Windows does not have a built-in support for .sh files. They need to install a third-party tool like Cygwin, Git bash or similar.